Updated to include -ForwardingAddress parameter email-forwarding.md (#1903)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/recipients/user-mailboxes/email-forwarding.md

Haven't used Exchange Management Shell much? Check out the [Exchange Management Shell](https://docs.microsoft.com/powershell/exchange/exchange-server/exchange-management-shell) topic to learn more. Take a look at the [Get-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/get-mailbox) and [Set-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/set-mailbox) topics for more details on the cmdlets used here.

> This example delivers email to the mailbox of Douglas Kohn and, at the same time, forwards all mail sent to Douglas Kohn to a external email addresses, douglaskohn.parents@fineartschool.net.

Reviewer: ```suggestion
This example delivers email to the mailbox of Douglas Kohn and also forwards all mail sent to Douglas Kohn to an external email address, douglaskohn.parents@fineartschool.net.
```
